rename.rb

```diff
@@ -31,16 +31,16 @@ def usage
 
 old_dirs = []
 Find.find('.') do |path|
-  next unless File.file?(path)
-  next if path =~ /\.git/
-  next if path == './rename.rb'
-
-  # Change content on all files
-  tmp_file = "#{path}.tmp"
-  system(%(sed 's/foreman_plugin_template/#{snake}/g' #{path} > #{tmp_file}))
-  system(%(sed 's/ForemanPluginTemplate/#{camel}/g' #{tmp_file} > #{path}))
-  system(%(sed 's/foremanPluginTemplate/#{camel_lower}/g' #{tmp_file} > #{path}))
-  system(%(rm #{tmp_file}))
+  if File.basename(path).start_with?('.')
```

Comment:
This means you'll never work on `.github`. You'll also ignore `.packit.yaml` and possibly others. Did you intend
```suggestion
  if File.basename(path) == '.git'
```

Then the comment on the next line is also redundant.

Reply:
No I don't, I totally forgot to check for other scenarios in there. Thanks for correcting it.